examples/chip-tool/templates/commands.zapt

```diff
@@ -97,9 +98,20 @@ static void OnInt64sAttributeResponse(void * context, int64_t value)
     command->SetCommandExitStatus(CHIP_NO_ERROR);
 }
 
-static void OnStringAttributeResponse(void * context, const chip::ByteSpan value)
+static void OnOctetStringAttributeResponse(void * context, const chip::ByteSpan value)
 {
-    ChipLogProgress(chipTool, "String attribute Response: %zu", value.size());
+    char buffer[CHIP_CONFIG_LOG_MESSAGE_MAX_SIZE];
+    if (CHIP_NO_ERROR == chip::Encoding::BytesToUppercaseHexString(value.data(), value.size(), &buffer[0], CHIP_CONFIG_LOG_MESSAGE_MAX_SIZE)) {
```

Comment:
We should consider logging something on error here (presumably because `value` is too big to fit?).  Maybe at least log the value size?